src/library/export/dlgcoverartcopy.cpp

```diff
@@ -0,0 +1,87 @@
+#include "library/export/dlgcoverartcopy.h"
+
+#include <QMessageBox>
+
+#include "library/library_prefs.h"
+#include "util/assert.h"
+
+DlgCoverArtCopy::DlgCoverArtCopy(QWidget* parent,
+        UserSettingsPointer pConfig,
+        CoverArtCopyWorker* worker)
+        : QDialog(parent),
+          Ui::CoverArtCopyDlg(),
+          m_pConfig(pConfig),
+          m_worker(worker) {
+    setupUi(this);
+    m_coverOverwritten = false;
+
+    setModal(true);
+
+    connect(m_worker,
+            &CoverArtCopyWorker::askOverwrite,
+            this,
+            &DlgCoverArtCopy::slotAskOverwrite);
+    connect(m_worker,
+            &CoverArtCopyWorker::canceled,
+            this,
+            &DlgCoverArtCopy::btnCancelClicked);
+}
+
+void DlgCoverArtCopy::btnCancelClicked() {
+    finish();
+}
+
+void DlgCoverArtCopy::showEvent(QShowEvent* event) {
+    QDialog::showEvent(event);
+    VERIFY_OR_DEBUG_ASSERT(m_worker) {
+        qDebug() << "m_worker did not initialized, about to crash";
```

Comment:
This comes up already as a critical errer.
```suggestion
```

Reply:
Should `kLogger.critical()` would work?

Reply:
Just remove the qDebug() VERIFY_OR_DEBUG_ASSERT has an internal logging that is sufficient.